administration/src/util/getDateFromUTC.ts

```diff
@@ -0,0 +1,2 @@
+const getDateFromUTC = (dateString: string): Date => new Date(`${dateString}Z`)
```

Comment:
It would be better if the GraphQL endpoint would send this `Z` suffix already. I believe changing from LocalDateTime to Instant should fix this as well.

Reply:
yes indeed i check this `Instant.now()` but couldn't figure out how to set this with `defaultExpression`